NebulaServer/Server.cs

```diff
@@ -99,5 +100,13 @@ private void OnVegeMinedUpdate(VegeMined packet, NebulaConnection conn)
             Player player = playerManager.GetPlayer(conn);
             playerManager.SendPacketToOtherPlayers(packet, player, DeliveryMethod.ReliableUnordered);
         }
+
+        private void OnPlayerColorChanged(PlayerColorChanged packet, NebulaConnection conn)
+		{
+            Player player = playerManager.GetPlayer(conn);
+            player.PlayerColor = packet.Color;
+            Console.WriteLine($"Player {player.Id} changed to color {packet.Color}");
```

Comment:
@hubastard Note to self, should add support of the Log class in the server code also to have the same API everywhere.